MultiCombobox: Add basic docs and make story public (#100806)
grafana/grafana: packages/grafana-ui/src/components/Combobox/MultiCombobox.mdx

import { Meta, Preview, ArgTypes } from '@storybook/blocks';

import { MultiCombobox } from './MultiCombobox';

<Meta title="MDX|MultiCombobox" component={MultiCombobox} />

# MultiCombobox

> The behavior of MultiCombobox is the same as Combobox, but it allows you to select multiple options. For all non-multi behavior, see the Combobox docs.

Reviewer: ```suggestion
The behavior of the MultiCombobox is similar to that of the Combobox, but it allows you to select multiple options. For all non-multi behaviors, see the Combobox documentation.
```

## "All" functionality

> Use the prop `enableAllOption` to show the "All" option.
> 
> If no filtering is done, it will select all options, returning the same array as the `options` prop.
> 
> With filtering, it will select the filtered subset of options.

Reviewer: ```suggestion
Use the prop `enableAllOption` to show the "All" option.

- If no filtering is applied, selecting "All" will select all options, returning the same array as the `options` prop.
- With filtering, selecting "All" will choose the filtered subset of options.
```
